README.md

```diff
@@ -33,24 +20,187 @@ Front end for Kano&#39;s authentication flow.
  
  If `assetsPath` is given the modal will look for a motif icon at `${assetsPath}/avatar/judoka-face.svg` and an arrow icon at `${assetsPath}/icons/link-arrow.svg` so you need to provide those files.
 
-## Installation
-Clone this repository.
-Run `bower i`
+## API
```

Comment:
Wow! This is really nicely documented! I would just raise one question: How confident are you that this will be updated here on the `README.md`? To be honest, I am doing all the documentation (only) on the actual code so goes straight to the docs generated by Polymer and it's easier to update/write. Is this here on the `README.md` duplicating the work?

Reply:
I wouldn't know how to do this in the comments. The readme gives you access to markdown.

Reply:
I guess we need a team or dev consensus on how which tool and place to put documentation.

Reply:
It's not comments, it's the documentation annotation: You can specify the purpose, how many, which type and names of the parameters and return types (if any)

https://www.polymer-project.org/2.0/docs/tools/documentation

Reply:
And you can use markdown as well.

Reply:
So we should decide if this is where we put the documentation. Does it allow anchor tags and such? More tightly tying us in to Polymer and its toolset does not fill me with joy/

Reply:
We are writing Polymer components: write Polymer documentation. That is actually in conformity with [Java docs](https://www.polymer-project.org/2.0/docs/tools/documentation#language) and [JSDocs](http://usejsdoc.org/about-getting-started.html).

I think it's already decided: me and @abridger are doing it.

Reply:
As described here: https://www.polymer-project.org/2.0/docs/tools/documentation

Reply:
and here: https://docs.google.com/document/d/1ryC8uwFAfukvMMIH9HN0pXDRjtdm7g8DBn2826p6FVc/edit#heading=h.9lbya0c3e0b8

Reply:
Seems decided then. What about non polymer code? JSDocs too?